codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -99,6 +103,12 @@ public void removeStream(Http2Stream stream) {
 
     @Override
     public void addListener(Listener listener) {
+        // Ensure we don't add the same listener twice.
```

Comment:
@Scottmitch @nmittler do we really need this or could we just allow "the same" listeners to be added multiple times?